packages/credential-provider-node/src/credential-provider-node.integ.spec.ts

```diff
@@ -118,7 +118,7 @@ jest.mock("@smithy/node-http-handler", () => {
               AccessKeyId: "CONTAINER_ACCESS_KEY",
               SecretAccessKey: "CONTAINER_SECRET_ACCESS_KEY",
               Token: "CONTAINER_TOKEN",
-              Expiration: "3000-01-01T00:00:00Z",
+              Expiration: new Date("3000/1/1").toISOString(),
```

Comment:
can you set all dates to an absolute value instead? this style will use the system time zone

Reply:
Updated all date strings to use fully specified offsets.